renku/core/commands/client.py

```diff
@@ -45,6 +45,7 @@ def pass_local_client(
     commit_only=None,
     ignore_std_streams=True,
     allow_empty=True,
+    do_not_commit_if_empty=False,
```

Comment:
Shouldn't this be defaulted to `True`? How do `allow_empty` and `do_not_commit_if_empty` differ?

Reply:
`allow_empty` raises an exception when it is `False` (and. nothing to commit) while `do_not_commit_if_empty` just returns with no exception. Both of these are used so I kept them. I'll rename them to `commit_empty` and `raise_if_empty` (which is considered when `commit_empty` is false) to make it a bit more clear.